test/packages/good/data_stream/foo/fields/some_fields.yml

```
@@ -35,3 +35,15 @@
 - name: error.message
   description: Error message.
   type: match_only_text
+- name: metric.*_bytes
```

Nit. Is this an intentional test case? We don't use to have wildcards mixed with suffixes, not sure if everything works as expected with this.
```suggestion
- name: metric.*.bytes
```

---

This was intentional to test the `pattern` specified for `name`. It really is an area of unspecified behavior at the moment. I was operating under the assumption that wildcards will be used to implement behavior similar to [`path_match`](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-templates.html#path-match-unmatch). But for now this is a test case just to check that the pattern matches fields like this:

https://github.com/elastic/integrations/blob/3c88555054fd24a461babebe624d8645b5feea6c/packages/aws/data_stream/cloudwatch_metrics/fields/package-fields.yml#L16

We should refine this area of the spec for dynamic fields.

---

I could remove the `pattern` for now. Then when we get to dynamic fields add it back to match the definition there.

---

Oh ok, lets leave it. Thanks.